### ISSUE TITLE
Application crashes when we click on any UI when AutomationProperties' "Font Size text box" is empty.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#324]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate Automation Properties from home page.
3. Navigate to text box present beside Font Size ,With backspace clear the value and then press tab or click on any other UI.

**Expected behavior**

Should be able to move the focus out of the text box by tab (or) Should be able to activate any other UI by clicking.

**Actual behavior**

Application gets crashed.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72557660-0e828880-3856-11ea-8fba-6b84a6b15947.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://dev.azure.com/microsoft/OS/_workitems/edit/24649879)

**Comment by chingucoding** [comment]
I thought that the NumberBox would make sure that the value would always be in the range. Does `x:Bind` get past this validation and it should not or is this by design?
